template/report.html.tmpl

```diff
@@ -1,441 +1,544 @@
+              <dd>{{ .Duration }}</dd>
+              <dt>Wait</dt>
+              <dd>{{ .Wait }}</dd>
+              <dt>Requests</dt>
+              <dd>{{ .Requests }}</dd>
+              <dt>Rate (as configured)</dt>
+              <dd>{{ .Rate }}</dd>
+              <dt>Throughput (as observed)</dt>
+              <dd>{{ .Throughput }}</dd>
+              <dt>Success</dt>
+              <dd>{{ .Success }}</dd>
+              {{ range $key, $value := .StatusCodes }}
+              <dt>Status Code {{ $key }}</dt>
+              <dd>{{ $value }}</dd>
+              {{ end }}
+            </dl>
+          </div>
+          {{- end }}
+        </details>
+        {{ end }}
+      </div>
+      <div>
+        <h3>Hardware Specification</h3>
         <dl>
-          <dt>Earliest</dt>
-          <dd>{{ .Earliest }}</dd>
-          <dt>Latest</dt>
-          <dd>{{ .Latest }}</dd>
-          <dt>End</dt>
-          <dd>{{ .End }}</dd>
-          <dt>Duration</dt>
-          <dd>{{ .Duration }}</dd>
-          <dt>Wait</dt>
-          <dd>{{ .Wait }}</dd>
-          <dt>Requests</dt>
-          <dd>{{ .Requests }}</dd>
-          <dt>Rate (as configured)</dt>
-          <dd>{{ .Rate }}</dd>
-          <dt>Throughput (as observed)</dt>
-          <dd>{{ .Throughput }}</dd>
-          <dt>Success</dt>
-          <dd>{{ .Success }}</dd>
-          {{ range $key, $value := .StatusCodes }}
-          <dt>Status Code {{ $key }}</dt>
-          <dd>{{ $value }}</dd>
-          {{ end }}
+          <div>
+            <dt>Computer</dt>
+            <dd>{{ "TODO ~ MacBook Pro (15-inch, 2019)" }}</dd>
+          </div>
+          <div>
+            <dt>Processor</dt>
+            <dd>{{ "TODO ~ 2.3 GHz 8-Core Intel Core i9" }}</dd>
+          </div>
+          <div>
+            <dt>Memory</dt>
+            <dd>{{ "TODO ~ 32 GB 2400 MHz DDR4" }}</dd>
+          </div>
+          <div>
+            <dt>Operating System</dt>
+            <dd>{{ "TODO ~ macOS Big Sur 11.5.2" }}</dd>
+          </div>
+          <div>
+            <dt>Docker</dt>
+            <dd>{{ "TODO ~ Docker Desktop 4.0.0.12 / 8 cores / 4 GB" }}</dd>
+          </div>
         </dl>
       </div>
-      {{- end }}
-    </details>
-    {{ end }}
-
-    <h3>Latencies</h3>
-    <table>
-      <tr>
-        <th>   </th>
-        <th>Total</th>
-        <th>Mean</th>
-        <th>50th</th>
-        <th>90th</th>
-        <th>95th</th>
-        <th>99th</th>
-        <th>Max</th>
-        <th>Min</th>
-      </tr>
+    </section>
+    <section>
+      <h2 id="latency">Latency</h2>
+      <div id="chart_div">Loading...</div>
+      <p>
+        Percentile range:
+        <input type="range" class="slider-width500" min="1" max="8" value="7" step="1" width="300px"
+          onchange="showValue(this.value)" />
+        <span id="percentileRange">99.99999%</span>
+      </p>
+      <table>
+        <tr>
+          <th> </th>
+          <th>Min</th>
+          <th>Mean</th>
```

Comment:
Maybe we do Mean before Min?

Reply:
It's more likely we'll remove it; we don't have neither mean nor 50th percentile in the design.